2024-09-13 14:31:02.006 [info] Extension host with pid 19071 started
2024-09-13 14:31:02.006 [info] Skipping acquiring lock for /home/zoro/Desktop/PROGRAMMING/MY-PROJECTS/EduAccess/default/User/workspaceStorage/0e890b5769eaa4e8de2ebda2087c0a26.
2024-09-13 14:31:02.120 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*'
2024-09-13 14:31:02.127 [info] ExtensionService#_doActivateExtension ms-python.python, startup: true, activationEvent: 'workspaceContains:manage.py'
2024-09-13 14:31:02.524 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2024-09-13 14:31:03.509 [info] Eager extensions activated
2024-09-13 14:31:03.523 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2024-09-13 14:31:03.526 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2024-09-13 14:31:03.630 [info] ExtensionService#_doActivateExtension ms-python.vscode-pylance, startup: false, activationEvent: 'api', root cause: ms-python.python
2024-09-13 14:31:16.728 [info] ExtensionService#_doActivateExtension vscode.git, startup: false, activationEvent: 'api', root cause: vscode.github
2024-09-13 14:31:16.873 [info] ExtensionService#_doActivateExtension cweijan.vscode-mysql-client2, startup: false, activationEvent: 'onStartupFinished'
2024-09-13 14:31:17.257 [info] ExtensionService#_doActivateExtension Postman.postman-for-vscode, startup: false, activationEvent: 'onStartupFinished'
2024-09-13 14:31:17.775 [error] Error: Starting server failed
	at P.$start (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2108511)
	at async P.sendNotification (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2093760)
2024-09-13 14:31:18.019 [warning] DiagnosticCollection with name 'settings' does already exist.
2024-09-13 14:35:48.184 [info] ExtensionService#_doActivateExtension ms-python.debugpy, startup: false, activationEvent: 'onLanguage:python'
2024-09-13 14:35:48.301 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2024-09-13 14:35:48.580 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:35:49.573 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:41:15.437 [info] ExtensionService#_doActivateExtension vscode.html-language-features, startup: false, activationEvent: 'onLanguage:html'
2024-09-13 14:43:18.939 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:43:19.942 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:43:44.646 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:43:45.647 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:48:18.585 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:48:19.590 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:04.538 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:05.538 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:30.872 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:31.872 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:34.145 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:51:35.147 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:52:32.677 [warning] DiagnosticCollection with name 'settings' does already exist.
2024-09-13 14:52:32.777 [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: false, activationEvent: 'onStartupFinished'
2024-09-13 14:52:33.916 [error] Error: ENOENT: no such file or directory, open '/home/zoro/Desktop/PROGRAMMING/MY-PROJECTS/EduAccess/default/User/globalStorage/vscodevim.vim/.registers'
2024-09-13 14:52:36.617 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:52:36.618 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 14:52:37.619 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:06:55.242 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:06:55.242 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:06:56.243 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:07:29.856 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:07:29.857 [error] Error: LanguageStatusItem with id 'python.analysis.typeCheckingMode' ALREADY exists
	at c.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:161:163942)
	at Object.createLanguageStatusItem (/snap/code/169/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:171:38483)
	at _0xbcc266.createStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:276048)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289607)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-13 15:07:30.857 [error] Error: Client is not running
	at P.sendRequest (/root/.vscode/extensions/ms-python.python-2024.14.1-linux-x64/out/client/extension.js:2:2091911)
	at Object._0x30e69b [as sendRequest] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:153560)
	at _0x18f5f4.getConfigInfo (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:278617)
	at _0xbcc266.updateStatusItem (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/extension.bundle.js:1:289680)
	at p (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792722)
	at E (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792929)
	at Timeout.I [as _onTimeout] (/root/.vscode/extensions/ms-python.vscode-pylance-2024.9.1/dist/vendor.bundle.js:2:1792831)
	at listOnTimeout (node:internal/timers:573:17)
	at process.processTimers (node:internal/timers:514:7)
2024-09-14 00:40:59.725 [info] Extension host terminating: renderer closed the MessagePort
2024-09-14 00:40:59.759 [error] Error: ENOENT: no such file or directory, open '/home/zoro/Desktop/PROGRAMMING/MY-PROJECTS/EduAccess/default/User/globalStorage/vscodevim.vim/.registers'
2024-09-14 00:40:59.764 [info] Extension host with pid 19071 exiting with code 0
